lib/WTSI/DNAP/Warehouse/Schema/Result/PacBioProductMetric.pm

```diff
@@ -72,6 +72,13 @@ PacBio run id, see 'pac_bio_run.id_pac_bio_tmp'
 
 Product id
 
+=head2 qc
```

Comment:
I'm not keen on the semantics of "qc", "qc_seq". It requires a little informed guessing to infer meaning. I would prefer something like "qc_pass", so you get queries like `SELECT id_pac_bio_product WHERE qc_pass is TRUE`.

That is unless you want to discourage uninformed use.

Reply:
re `qc*` semantics - yes, they could be better - but consistency with the existing use in Illumina world is important (and I'd argue more so). Do they match Illumina well? Or is it a somewhat forced alignment?